docs/docs/testing-your-assistant.mdx

```diff
@@ -117,113 +117,138 @@ Here are some examples:
+[recall](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html) and
+[f1-score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) for each intent,
 as well as providing an overall average. You can save these reports as JSON files using the `--report` argument.
 
-The confusion matrix shows you which intents are mistaken for others;
-any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
+The confusion matrix shows which intents are mistaken for others.
+Any samples which have been incorrectly predicted are logged and saved to a file called `errors.json` for easier debugging.
 
-The histogram that the script produces allows you to visualise the confidence distribution for all predictions, with the volume of correct and incorrect predictions being displayed by blue and red bars respectively. Improving the quality of your training data will move the blue histogram bars to the right and the red histogram bars to the left of the plot.
+The histogram that the script produces allows you to visualize the confidence distribution for all predictions,
+with the volume of correct and incorrect predictions being displayed by blue and red bars respectively.
+Improving the quality of your training data will move the blue histogram bars to the right and the
+red histogram bars to the left of the plot.
 
-### Response Selection
+#### Response Selectors
 
-The evaluation script will produce a combined report for all response selector models in your pipeline.
+Similar to intent classifiers, the evaluation script outputs the same set of artifacts for response selectors -
+report, confusion matrix, confidence histogram and errors.
+If you have multiple response selectors in your pipeline, the evaluation for all of them is combined in a single report.
 
 The report logs precision, recall and f1 measure for
-each response, as well as providing an overall average.
+each sub-intent of a [retrieval intent](./glossary.mdx#retrieval-intent), as well as providing an overall average.
 You can save these reports as JSON files using the `--report` argument.
 
-### Entity Extraction
+#### Entity Extraction
 
-Only trainable entity extractors, such as the `DIETCLassifier` and `CRFEntityExtractor` will be
-evaluated by the `rasa test` script. If you pretrained extractors like the `DucklingHTTPExtractor`
-Rasa Open Source will not include these in the evaluation.
-
-`rasa test` reports recall, precision, and f1 measure for each entity type that
+`rasa test` also reports recall, precision, and f1-score for each entity type that
 your trainable entity extractors are trained to recognize.
 
+Only trainable entity extractors, such as the `DIETClassifier` and `CRFEntityExtractor` will be
+evaluated by the `rasa test` script. If you use pretrained extractors like the `DucklingHTTPExtractor`,
```

Comment:
evaluated by `rasa test`. Pretrained extractors like the `DucklingHTTPExtractor` are not evaluated.

Reply:
Just saw this. Please ignore my comment above.